console-subscriber/src/builder.rs

```diff
@@ -268,6 +274,24 @@ impl Builder {
         Self { self_trace, ..self }
     }
 
+    #[cfg(feature = "grpc-web")]
+    /// Sets whether to enable the grpc-web support.
+    ///
+    /// By default, this is `false`. If enabled, the console subscriber will
+    /// serve the gRPC-Web protocol in addition to the standard gRPC protocol.
+    /// This is useful for serving the console subscriber to web clients.
+    ///
+    /// [serve_with_grpc_web] is used to provide more advanced configuration
+    /// for the gRPC-Web server.
+    ///
+    /// [serve_with_grpc_web]: crate::Server::serve_with_grpc_web
```

Comment:
Use monospace for function names:
```suggestion
    /// [`serve_with_grpc_web`] is used to provide more advanced configuration
    /// for the gRPC-Web server.
    ///
    /// [`serve_with_grpc_web`]: crate::Server::serve_with_grpc_web
```